src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -463,6 +467,46 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
         }
       }
 
+    /**
+      * If the sink is a PostgresDB with Timescale extension
+      * creates hypertable, with a partitioning column.
+      */
+    if (product == Postgresql && isTimescale) {
+      val timeColumn: String   = config.getString(pfx("table.time.column"))
+      val chunkTimeInterval: String = config
```

Comment:
I'd move these parameters to the top level sink config class scope so they are available to subclasses.